fixcoveragefilepaths.py

```diff
@@ -0,0 +1,67 @@
+import xml.etree.ElementTree as xml
+import glob, os, re, sys, getopt
```

Comment:
Needn't necessarily address it now but [argparse](https://docs.python.org/2/howto/argparse.html) is the idiomatic and the recommended way to do cmdline option parsing.